widgetry/src/widgets/button.rs

```diff
@@ -271,6 +308,507 @@ impl Btn {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct ButtonBuilder<'a> {
+    padding: EdgeInsets,
+    stack_spacing: f64,
+    hotkey: Option<MultiKey>,
+    tooltip: Option<Text>,
+    stack_axis: Option<geom_batch_stack::Axis>,
+    is_label_before_image: bool,
```

Comment:
What's the criteria for putting a field here vs in `ButtonStyle`? If all 3 button styles share it? (I started thinking about `is_label_before_image`, but it would also apply to `padding`, `stack_spacing`, `stack_axis`)

Reply:
I think the name is fine for now; thanks for the explanation